src/scripts.js

```diff
@@ -387,3 +392,20 @@ function findRecipesWithCheckedIngredients(selected) {
     }
   })
 }
+
+// function removePantryIngredients(itemToRemove) {
+//   let letIte
```

Comment:
I'm interested into seeing how this will effect our pantry. I'm still not 100% sure how to format it, but we'll get it